gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Sep 26 23:06:06 MSK 2016
+#Sun Oct 23 22:01:18 JST 2016
```

Comment:
The `gradle-wrapper.properties` file might be conflict.

You could accept yours.